css/styles.scss

```diff
@@ -43,6 +43,8 @@
 	.notification-wrapper {
 		display: flex;
 		flex-direction: column;
+		margin-bottom: 44px;
+		max-height: calc(100vh - 50px * 4 - 44px) !important;
```

Comment:
lets hope no one uses a device that is smaller than 244 pixels + 50px of the header